yaksh/models.py

```diff
@@ -150,6 +156,23 @@ def get_teachers(self):
     def remove_teachers(self, *teachers):
         self.teachers.remove(*teachers)
 
+    def create_demo(self, user):
+        course = Course.objects.filter(creator=user, name="Yaksh Demo course")
+        if not course:
+            course, c_status= Course.objects.get_or_create(name="Yaksh Demo course",
```

Comment:
Since you are already checking if a course exists, just do a "create" over here.